src/main/java/com/linkedin/kafka/clients/common/config/LiAbstractConfig.java

```diff
@@ -0,0 +1,35 @@
+package com.linkedin.kafka.clients.common.config;
+
+import org.apache.kafka.common.KafkaException;
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.utils.Utils;
+
+import java.util.Map;
+
+public abstract class LiAbstractConfig extends AbstractConfig {
```

Comment:
Agree, but I didn't want to duplicate ```getInstance(String key, Class<T> t)``` implementation in ```LiKafkaProducerConfig``` and ```LiKafkaConsumerConfig``` classes. We can remove it once we agree on the approach above.